RELEASE.md

```diff
@@ -6,6 +6,10 @@
    1. saxon: 9.8.0-4 → 9.8.0-8
    1. wadl-tools: 1.0.37 → 1.0.38
 1. Clean up : Added simple tests to confirm case of a header param name is irrelevant.
+1. In order to help debugging, checker metadata can now be displayed or logged in a human friendly format by:
+   1. Supplying a ```StreamResult``` to capture the metadata when using checker/validator APIs.
+   1. Using ```-O``` in CLI utilities.
+   1. Setting the logger to ```TRACE``` level.
```

Comment:
hmm.  I just assumed folks could simple set the root logger to TRACE to get it and they wouldn't have to know thot set WADLCheckerBuilder to log. 

If someone is interested in capturing the output outside of a log, they can set a StreamResult, which is what happens when you do a `-O` in the cli utils.

Reply:
...in other words.  If you just want the TRACE for the metadata -- it's better to just pass a stream result and capture it that way.  Make sense?

Reply:
So Repose should send a Stream Result instead of just setting that specific logger to `TRACE`?

I can see someone wanting that information, but not wanting to put all of Repose and all of Repose's dependencies in a mode that was logging at the `TRACE` level. That is what would happen if the root logger for Repose was set to the `TRACE` level.

Reply:
I debated this with myself and where's where I ended up.  This isn't really like `api-coverage-logger` because that's a true log, it changes over time.  This is info about what just loaded.  The info is always made available via JMX so at anytime you can bind to the process and pull that metadata and look at exactly what loaded and with what options.  I'd rather the trace cover the entire loading process and you get that via setting trace at `com.rackspace.com.papi.components.checker.wadl.WADLCheckerBuilder` which is responsible for loading the WADL. 

I don't think you have to add a feature to Repose, in the cases where you need to look at the metadata you can always do so via JMX.  The other use case is via the CLI tools and for that I created the `-O` option.

Reply:
Sounds good.